widgetssdk/src/main/java/com/glia/widgets/core/visitor/GliaVisitorInfo.java

```diff
@@ -5,12 +5,16 @@
 
 import java.util.Map;
 
+/**
+ * @deprecated since 1.9.0
+ */
+@Deprecated
 public class GliaVisitorInfo implements VisitorInfo {
     public String name;
     public String email;
     public String phone;
     public String note;
-    public LinkedTreeMap<String, String> customAttributes;
+    public Map<String, String> customAttributes;
```

Comment:
perhaps I missed something, but it looks like breaking changes since you have changed property type. Is it?
Could we avoid any breaking changes and just deprecate this structure?